services/azure-devops/azure-devops-artifacts.service.js

```diff
@@ -0,0 +1,102 @@
+'use strict'
+
+const Joi = require('joi')
+const { version: versionColor } = require('../color-formatters')
+const { addv } = require('../text-formatters')
+const {
+  renderVersionBadge: renderNugetVersionBadge,
+} = require('../nuget/nuget-helpers')
+const { NotFound } = require('..')
+const AzureDevOpsBase = require('./azure-devops-base')
+const { keywords } = require('./azure-devops-helpers')
+
+const getPackagesSchema = Joi.object({
+  count: Joi.number().required(),
+  value: Joi.array()
+    .items(
+      Joi.object({
+        id: Joi.string().required(),
+        normalizedName: Joi.string().required(),
+        protocolType: Joi.string().required(),
+        versions: Joi.array()
+          .items(
+            Joi.object({
+              version: Joi.string().required(),
+            })
+          )
+          .required()
+          .min(1),
+      })
+    )
+    .required(),
+}).required()
+
+function renderArtifactBadge(version, protocolType) {
+  return protocolType === 'NuGet'
+    ? renderNugetVersionBadge({ version, feed: `Azure Artifacts NuGet` })
+    : {
+        label: `Azure Artifacts ${protocolType}`,
+        message: addv(version),
+        color: versionColor(version),
+      }
```

Comment:
We'll need to make some adjustments to the label/feed here, though not entire sure precisely what they should be just yet. One of the core tenants of our [specification that drives the badges](https://github.com/badges/shields/blob/master/spec/SPECIFICATION.md) is that the badges be non-promotional (i.e. labels should not focus on service names).

This is why the other badges that display data from Azure DevOps have labels like `build`, `deployment`, `tests`, and `coverage` as opposed to things like `Azure Pipelines`.

However, there's plenty of existing version badges, and certainly other nuget-version badges that don't seem to be following this.

I suspect that at a minimum we'll need to:
* use lower case for the feed name in the nuget case
* skip the custom label for the non-nuget case (leave it as the default `version`)

this would at least ensure that these nuget badges are consistent with the other nuget version badges, and non-nuget packages (e.g. npm packages in in AzDO artifact feed) would be more consistent with other non-nuget version badges 

cc @badges/shields-core for other perspectives/in case i'm forgetting anything on our nuget badge labels/etc.